app/views/account/details.scala.html

```diff
@@ -4,28 +4,28 @@
     subscriberId: Option[String]
 )(implicit request: RequestHeader, flash: Flash, touchpointBackendResolution: services.TouchpointBackend.Resolution)
 
-@main("Cancel your papers when you're away | The Guardian", edition = UK, touchpointBackendResolutionOpt = Some(touchpointBackendResolution)) {
+@main("Manage your subscription | The Guardian", edition = UK, touchpointBackendResolutionOpt = Some(touchpointBackendResolution)) {
```

Comment:
Just a reminder to put this change into a separate PR, and so we can discuss what kind of multi-product copy we need, with Julia and maybe the Commercial team. This is a great starting point though.